### DISCUSSION TITLE
metabolicTasks Prepare the metabolic task list

### DISCUSSION THREAD

**Discussion by helenaherand** [SysBioChalmers/Human-GEM#373] [Q&A] [answered]
I found https://github.com/SysBioChalmers/Human-GEM/blob/main/data/README.md that contains metabolic task files used for evaluating the functionality of Human-GEM.

Despite https://sysbiochalmers.github.io/Human-GEM-guide/gem_extraction/ with uses Matlab, is it possible load the list of metabolic tasks in python3?

**Comment by JonathanRob** [comment, marked as answer]
Hi @helenaherand, the functions used to load and utilize the task files are all written in Matlab (in the [RAVEN](https://github.com/SysBioChalmers/RAVEN) package); unfortunately I do not yet know of a python package than can read and use these files. But it would of course be super nice to have.

**Comment by JonathanRob** [comment]
I think this Issue is better suited as a Discussion, so I will convert it now.

**Comment by PkiwiBird** [comment]
Hi @helenaherand. Do you still need it? I can share a script for that but it converts to a python list where each task is in the format to be dealt with [troppo](https://github.com/BioSystemsUM/troppo) package. Not sure if it's what you want.
